AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/MutationSync/OutgoingMutationQueue/SyncMutationToCloudOperation.swift

```diff
@@ -82,11 +95,21 @@ class SyncMutationToCloudOperation: Operation {
         } catch {
             let apiError = APIError.unknown("Couldn't decode model", "", error)
             finish(result: .failed(apiError))
-            return
+            return nil
         }
+        return request
+    }
 
-        log.verbose("\(#function) sending mutation with sync data: \(request)")
-        mutationOperation = api.mutate(request: request) { asyncEvent in
+    func makeAPIRequest(_ apiRequest: GraphQLRequest<MutationSync<AnyModel>>) {
+        guard let api = api else {
+            // TODO: This should be part of our error handling routines
+            log.error("\(#function): API unexpectedly nil")
+            let apiError = APIError.unknown("API unexpectedly nil", "")
```

Comment:
Consider the "nil API" extension of DataStoreError (the actual symbol escapes me for the moment)

Reply:
Tried using this:
```
            let nilAPIError = DataStoreError.nilAPIHandle()
```
but using `.failed` is expecting an APIError, not a Datastore error... leaving this for now.